### ISSUE TITLE
Object rotation bugged in TileMap

### ISSUE THREAD

**Issue by bunny-therapist** [pythonarcade/arcade#928]
## Bug Report

Objects with rotation in tilemap object layer are positioned incorrectly.

I suspect that this TileMap code is bugged:
```
rotated_center_x = center_x * cos_rotation - center_y * sin_rotation
rotated_center_y = center_y * sin_rotation + center_y * cos_rotation
```
My object looked fine if I replace the first center_y in rotated_center_y with center_x.

### Actual behavior:
Rotated object should appear offset from their position in Tiled.

### Expected behavior:
Rotated object should appear as it does in Tiled.

### Steps to reproduce/example code:
Create a object in Tiled with width > height, give it rotation = 90. Load into TileMap and view.

**Comment by pvcraven** [comment]
What version of Arcade?

**Comment by bunny-therapist** [comment]
I pulled the development branch.

This is basically a typo in the rotation matrix.

**Comment by bunny-therapist** [comment]
arcade @ git+https://github.com/pythonarcade/arcade.git@dce01b0b47b642a1535005d971700d7a296ea904

**Comment by pvcraven** [comment]
Thanks. That took me a while to replicate.

**Comment by pvcraven** [comment]
<img width="479" alt="4" src="https://user-images.githubusercontent.com/1463116/130368520-a5e40931-bf78-4f1c-987f-77590db830d6.png">

**Comment by bunny-therapist** [comment]
Is this showing tile editor and in-game view of a vertical rocket rotated 90 degrees, after fixing the bug? If so, nice!

**Comment by bunny-therapist** [comment]
Oh, wait, no, I see now. This shows the bug in action. I missed that it was not aligned with the surroundings.

Yes, that's exactly why I had. Object was higher up than it should because the transformation is wrong.

**Comment by pvcraven** [comment]
Right. Tiled is on the bottom, and running game is on the top with the rocket not positioned correctly.

**Comment by pvcraven** [comment]
Fixed in dev stream. Thanks for the report.

**Comment by Cleptomania** [comment]
Released with 2.6.0
